src/components/Products.vue

```diff
@@ -86,6 +87,9 @@ export default {
     addToCart2(e, payload) {
       this.addToCart(payload);
     },
+    delFromCart2(e, payload) {
+      this.delFromCart(payload);
+    },
```

Comment:
Tutaj bedzie to trzeba przemianować na pojedynczą metodę ktora umożliwi dodwanie wielu produktow do koszyka. 
Pomogę Ci z tym jak co